databricks/koalas/series.py

```diff
@@ -1003,6 +1003,38 @@ def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, d
         kdf._metadata = Metadata(data_columns=[self.name], index_map=[(index_name, None)])
         return _col(kdf)
 
+    def sort_values(self, ascending: bool = True, inplace: bool = False,
+                    na_position: str = 'last') -> Union['Series', None]:
+        """
+        Sort by the values.
+
+        Sort a Series in ascending or descending order by some criterion.
+
+        Parameters
+        ----------
+        ascending : bool or list of bool, default True
+             Sort ascending vs. descending. Specify list for multiple sort
+             orders.  If this is a list of bools, must match the length of
+             the by.
+        inplace : bool, default False
+             if True, perform operation in-place
+        na_position : {'first', 'last'}, default 'last'
+             `first` puts NaNs at the beginning, `last` puts NaNs at the end
+
+        Returns
+        -------
+        sorted_obj : Series ordered by values.
+        """
```

Comment:
Eh, why don't we add the doctests as examples as well? (https://github.com/pandas-dev/pandas/blob/v0.24.2/pandas/core/series.py#L2668-L2831)

Reply:
Good point. I've added them with fa32e67.